frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -162,7 +170,14 @@ export const CreateOCS = withHandlePromise<CreateOCSProps & HandlePromiseProps>(
             <SelectNodesSection
               table={AttachedDevicesNodeTable}
               customData={{ filteredNodes, nodes, setNodes }}
-            />
+            >
+              {isMinimal && (
+                <div className="ceph-ocs-install__minimal-msg">
+                  Since the selected nodes do not satisfy the minimum requirements of 16 CPUs and 64
+                  GiB of RAM per node, a minimal cluster will be deployed.
```

Comment:
As we don't know whether minimal will be also deployed or not.

I think we should communicate that we will try to deploy the minimal cluster with the specs provided.